pandas/core/strings.py

```diff
@@ -1842,23 +1842,45 @@ def extractall(self, pat, flags=0):
         return str_extractall(self._orig, pat, flags=flags)
 
     _shared_docs['find'] = ("""
+    Return the lowest index of the substring.
+
     Return %(side)s indexes in each strings in the Series/Index
     where the substring is fully contained between [start:end].
     Return -1 on failure. Equivalent to standard :meth:`str.%(method)s`.
 
     Parameters
     ----------
     sub : str
-        Substring being searched
+        Substring being searched.
     start : int
-        Left edge index
+        Left edge index.
     end : int
-        Right edge index
+        Right edge index.
 
     Returns
     -------
     found : Series/Index of integer values
 
+    Examples
+    --------
+    >>> s = pd.Series('Finding with pandas')
+    >>> s.str.find('wi')
+    0    8
+    dtype: int64
+
+    Return -1 if the substring is not found.
+
+    >>> s.str.find('wi', start=0, end=5)
+    0   -1
+    dtype: int64
+
+    >>> s = pd.Series(['get pandas','Did you find it','i think you can find it'])
```

Comment:
This line is too long. Can you fix it by wrapping? Start the next line with `...` and indent appropriately

Reply:
Done.